tox_packet/src/dht/crypto_handshake.rs

```diff
@@ -126,7 +130,7 @@ pub struct CryptoHandshakePayload {
     /// used to make sure that possible attacker can't combine payload from old
     /// `CryptoHandshake` with new `Cookie` and try to do mess sending such
     /// packets.
-    pub cookie_hash: sha512::Digest,
+    pub cookie_hash: [u8; <Sha512 as Digest>::OutputSize::USIZE],
```

Comment:
PingId?

Reply:
Nope, It's different type which uses sha512.